solutions/go/match_alphanumeric/explanation.md

```diff
@@ -0,0 +1,21 @@
+The entry point for your grep implementation is in `cmd/mygrep/main.go`.
```

Comment:
@nikandfor I don't think this explanation file should be here - this is for the first stage, no? Same for other stages - the explanation needs to be empty (until we get technical writers to come and write them)

Reply:
I copied all the stage folder, didn't noticed that file. Will remove it.
Also checked git, there is definition.yml in each stage. Is it ok?

Reply:
@nikandfor yep, `definition.yml` is fine!